adapters/handlers/rest/configure_api.go

```diff
@@ -344,37 +350,46 @@ func registerModules(appState *state.State) error {
 		}
 	}
 
-	modulesDir := appState.ServerConfig.Config.ModulesPath
-	if len(modulesDir) == 0 {
-		modulesDir = "./modules"
-	}
-	files, err := ioutil.ReadDir(modulesDir)
-	if err != nil {
-		return errors.Wrapf(err, "cannot read modules from directory: %s", modulesDir)
+	// LOADING modules is disabled, as for now we have decided against using Go
+	// plugins for modules.
+
+	// modulesDir := appState.ServerConfig.Config.ModulesPath
```

Comment:
I would remove the whole go plugins implementation code, I think that we will not get back to it any time soon

Reply:
You're probably right. Will do!

Reply:
removed.